src/main/java/be/seeseemelk/mockbukkit/inventory/InventoryMock.java

```diff
@@ -29,12 +29,24 @@
 
 	public InventoryMock(InventoryHolder holder, int size, InventoryType type)
 	{
+		assertTrue(
+				"Size for custom inventory must be a multiple of 9 between 9 and 54 slots (got " + size + ")",
+				9 <= size && size <= 54 && size % 9 == 0);
```

Comment:
I think a Validate would be more appropriate here.

Especially because this is what the Bukkit APi does:
`Validate.isTrue(9 <= size && size <= 54 && size % 9 == 0, "Size for custom inventory must be a multiple of 9 between 9 and 54 slots (got " + size + ")");`

Almost an exact copy.

```suggestion
		Validate.isTrue(9 <= size && size <= 54 && size % 9 == 0,
				"Size for custom inventory must be a multiple of 9 between 9 and 54 slots (got " + size + ")");
```